uploadNeuroDB/imaging_non_minc_insertion.pl

```diff
@@ -461,7 +461,7 @@ =head2 Methods
 }
 
 # determine the session ID
-my ($session_id, $requiresStaging) = NeuroDB::MRI::getSessionID(
+my ($session_id) = NeuroDB::MRI::getSessionID(
```

Comment:
I think `NeuroDB::MRI::getSessionID` requires 5 arguments. You only supply 4...Might happen elsewhere in the code too.